component.js

```diff
@@ -19,6 +23,12 @@ function Component(name, parent, _proto, extend) {
 
 module.exports = Component;
 
+function isDefinedProperty(prop) {
+  var isDataProp = 'value' in prop || 'writable' in prop;
+  var isAccessProp = 'get' in prop || 'set' in prop;
+  return isDataProp || isAccessProp;
```

Comment:
Whew. Yes, I do think that disallowing the value property for objects is a valid design decision. XD